### PR TITLE
Add an option to ignore private teams to peribolos

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

/assign @cjwagner @fejta

### CODE REVIEW

**Comment by cblecker** [comment]
Just a usability question.. I'm assuming you're doing this to hide certain teams from public view? If you need that level of privacy, wouldn't it just make sense to make your peribolos config in a private repo?

**Comment by stevekuznetsov** [comment]
Maybe, but if config is managed in a central repo already it's not going to be nice UX to fragment that, either. I can filter the teams myself (if the lack of a team in the config does not make it removed on sync) as well

File: prow/cmd/peribolos/main.go [hunk 1]
```diff
@@ -80,6 +81,7 @@ func (o *options) parseArgs(flags *flag.FlagSet, args []string) error {
 	flags.IntVar(&o.tokensPerHour, "tokens", defaultTokens, "Throttle hourly token consumption (0 to disable)")
 	flags.IntVar(&o.tokenBurst, "token-burst", defaultBurst, "Allow consuming a subset of hourly tokens in a short burst")
 	flags.StringVar(&o.dump, "dump", "", "Output current config of this org if set")
+	flags.BoolVar(&o.ignorePrivateTeams, "ignore-private-teams", false, "Do not dump private teams if set")
```
Comment (fejta) [inline prow/cmd/peribolos/main.go]: Won't this cause peribolos to then delete the private teams once it starts managing teams?

**Comment by stevekuznetsov** [comment]
@fejta yeah, sorry -- misread the way that worked. Added the other half of this functionality

File: prow/cmd/peribolos/main.go [hunk 2]
```diff
@@ -505,6 +510,10 @@ func configureTeams(client teamClient, orgName string, orgConfig org.Config, max
 		return nil, fmt.Errorf("failed to list teams: %v", err)
 	}
 	for _, t := range teamList {
+		p := org.Privacy(t.Privacy)
```
Comment (fejta) [inline prow/cmd/peribolos/main.go]: combine with next line

**Comment by nikhita** [comment]
Hmm, why is the "LGTM has been added" comment added twice here: https://github.com/kubernetes/test-infra/pull/12054#issuecomment-479722387 and https://github.com/kubernetes/test-infra/pull/12054#issuecomment-479742930?

Also saw this on https://github.com/kubernetes/test-infra/pull/9387#issuecomment-479206680, but it doesn't appear to be consistent... :thinking:

**Comment by cjwagner** [comment]
> Hmm, why is the "LGTM has been added" comment added twice here: [#12054 (comment)](https://github.com/kubernetes/test-infra/pull/12054#issuecomment-479722387) and [#12054 (comment)](https://github.com/kubernetes/test-infra/pull/12054#issuecomment-479742930)?
> 
> Also saw this on [#9387 (comment)](https://github.com/kubernetes/test-infra/pull/9387#issuecomment-479206680), but it doesn't appear to be consistent...

Thanks for pointing this out. I PRed a fix: https://github.com/kubernetes/test-infra/pull/12075

**Pull request merged**